lib/easybill-rest-client/api_client.rb

```diff
@@ -31,7 +33,12 @@ def request(method, endpoint, params = {})
         sleep: retry_cool_off_time,
         on: EasybillRestClient::TooManyRequests
       ) do
-        response = perform_request(method, endpoint, params)
+        response = begin
+                     perform_request(method, endpoint, params)
+                   rescue Net::OpenTimeout
+                     logger.warn("Unable to open connection after #{OPEN_TIMEOUT}s, retrying...")
+                     retry
```

Comment:
So if easybill is down, this will be an infinite loop? Shouldn't we abort and fail at some point?